exact_solvers/acoustics_demos.py

```diff
@@ -73,9 +79,12 @@ def char_solution_interactive():
 
     mainwidget = interact(char_solution, t=twidget, rho=rhowidget, K=Kwidget);
 
-    mainwidget.widget.close()
-    display(interact_gui)
-    display(mainwidget.widget.out)
+    try:
```

Comment:
I guess the reason for putting this in a try/except block is that when it is run in a certain mode then this code raises an exception and doesn't need to be run.  Doing this has gotten me into trouble in the past; it would be nice to catch any exceptions other than the one we expect by writing `except <particular exception type>:`.

Reply:
I don't remember now if I added this or why.  It's fine to improve the code to catch the exception if you want.